packages/archi-lwt/archi-lwt.0.1.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "Antonio Monteiro <anmonteiro@gmail.com>"
+authors: [ "Antonio Monteiro <anmonteiro@gmail.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/anmonteiro/archi"
+bug-reports: "https://github.com/anmonteiro/archi/issues"
+dev-repo: "git+https://github.com/anmonteiro/archi.git"
+doc: "https://anmonteiro.github.io/archi/"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune" {>= "1.0"}
+  "archi"
```

Comment:
Are the releases of the subpackages going to be linked all together? If so I'd highly recommand to use `{= version}` here